diff -urN rrdtool-1.0.50.org/contrib/php4/rrdtool.c rrdtool-1.0.50/contrib/php4/rrdtool.c
--- rrdtool-1.0.50.org/contrib/php4/rrdtool.c	2003-06-30 10:47:13.000000000 +0200
+++ rrdtool-1.0.50/contrib/php4/rrdtool.c	2005-10-14 22:06:35.000000000 +0200
@@ -274,6 +274,7 @@
 	zval *p_calcpr;
 	HashTable *args_arr;
 	int i, xsize, ysize, argc;
+	double ymin, ymax;
 	char **argv, **calcpr;
     
 
@@ -321,7 +322,7 @@
 		}
    
 		optind = 0; opterr = 0; 
-		if ( rrd_graph(argc-1, &argv[1], &calcpr, &xsize, &ysize) != -1 )
+		if ( rrd_graph(argc-1, &argv[1], &calcpr, &xsize, &ysize, NULL, &ymin, &ymax) != -1 )
 		{
 			array_init(return_value);
 			add_assoc_long(return_value, "xsize", xsize);
